recipes/cron.rb

```diff
@@ -46,11 +46,13 @@ class ::Chef::Recipe
     source "#{dist_dir}/init.d/chef-client.erb"
     mode 0755
     variables(client_bin: client_bin)
+    not_if { node['platform_family'] == 'freebsd'}
```

Comment:
that should probably not be a not_if and should wrap the template completely so it doesn't even appear as a resource on freebsd.

foodcritic might get annoyed by this, but foodcritic can kinda suck it and you can use `# ~FCXYZ` to turn it off.